sceptre/scipool/templates/sc-product-lambda.yaml

```diff
@@ -0,0 +1,64 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Lambda ServiceCatalog product
+Parameters:
+  RepoRootURL:
+    Type: String
+    Description: Root url for the repo containing the product templates.
+  ProductName:
+    Type: String
+    Description: Name of the product that will be visible to the end user
+    Default: 'Lambda: Docker Runner'
+  StackDatetime:
+    Type: String
+    Description: Last updated date and time of product
+    Default: ''
+  ReplaceProvisioningArtifacts:
+    Type: String
+    Description: "Whether to keep or replace the provisioning artifact identifier on update"
+    Default: 'false'
+Resources:
+  LambdaProduct:
+    Type: AWS::ServiceCatalog::CloudFormationProduct
+    Metadata:
+      cfn-lint:
+        config:
+          ignore_checks:
+            - E3002
+            - E3003
+            - W2001
```

Comment:
Why are these lint checks ignored?

Reply:
I'm not sure anymore.  i copied from the EC2 product yaml file. will double check if they are still needed.

Reply:
W2001 was a cfn-lint bug that is now fixed.

These are still cfn-lint bugs:
* E3003 Property Owner missing at Resources/scec2linuxproduct/Properties
* E3002 Invalid Property Resources/scec2linuxproduct/Properties/Fn::Transform

I've updated all references to these in PR #104